website/homepage/src/docs/__init__.py

```diff
@@ -79,6 +80,7 @@ def override_signature(name, signature):
 override_signature("Tabs", "with gradio.Tabs():")
 override_signature("Group", "with gradio.Group():")
 override_signature("Box", "with gradio.Box():")
+override_signature("Dataset", "gr.Dataset(components, samples)")
```

Comment:
I think since `components` and `samples` are not positional args, they wouldn't show up with the current logic.

Seems like a one-off so probably better to override the signature manually